### PR TITLE
Make permutation and transpose be views

### PR DESCRIPTION
**Pull request by asuhan**

### CODE REVIEW

File: torch_xla/csrc/ops/ops.cpp [hunk 1]
```diff
@@ -137,15 +137,22 @@ NodePtr ReluOp(const Value& input) {
                    std::move(lower_fn));
 }
 
-NodePtr TransposeOp(const Value& input, xla::int64 dim0, xla::int64 dim1) {
-  xla::int64 rank = input.shape().rank();
+std::vector<xla::int64> MakeTransposePermutation(xla::int64 dim0,
```
Comment (dlibenzi) [inline torch_xla/csrc/ops/ops.cpp]: This should go in helpers.*

Comment (asuhan) [reply torch_xla/csrc/ops/ops.cpp]: Done.

File: torch_xla/csrc/helpers.cpp [hunk 2]
```diff
@@ -205,6 +205,18 @@ xla::XlaOp XlaHelpers::ReshapeToRank(const xla::XlaOp& input,
   return xla::Reshape(input, dimensions);
 }
 
+std::vector<xla::int64> XlaHelpers::MakeTransposePermutation(xla::int64 dim0,
+                                                             xla::int64 dim1,
+                                                             xla::int64 rank) {
+  xla::int64 canonical_dim0 =
+      XlaHelpers::GetCanonicalDimensionIndex(dim0, rank);
```
Comment (dlibenzi) [inline torch_xla/csrc/helpers.cpp]: No need for XlaHelpers:: ... we are in XlaHelpers now.

Comment (asuhan) [reply torch_xla/csrc/helpers.cpp]: Done.

**Comment by asuhan** [comment]
Build is broken because of a core PyTorch change, I'll fix it separately.

**Pull request merged**